test/unit/mdc-auto-init/mdc-auto-init.test.js

```diff
@@ -18,6 +18,7 @@ import bel from 'bel';
 import td from 'testdouble';
 import {assert} from 'chai';
 import mdcAutoInit from '../../../packages/mdc-auto-init';
+import {emit} from '../../../packages/mdc-auto-init';
```

Comment:
We can proxy over `mdcAutoInit ` to test private method `emit` is called, so we could remove this line.